wcivf/apps/elections/filters.py

```diff
@@ -0,0 +1,94 @@
+from urllib.parse import urlencode
+from django.db.models import BLANK_CHOICE_DASH
+from django.utils.encoding import force_str
+from django.utils.safestring import mark_safe
+from django_filters.widgets import LinkWidget
+
+import django_filters
+
+from elections.models import Election
+
+
+def _get_election_types_choices_for_qs(qs):
+    return [
+        ("europarl", "European Parliament"),
+        ("gla", "London Assembly"),
+        ("local", "Local"),
+        ("mayor", "Mayoral"),
+        ("pcc", "Police and Crime Commissioner"),
+        ("nia", "Northern Irish Assembly"),
+        ("ref", "Referendum"),
+        ("sp", "Scottish Parliament"),
+        ("senedd", "Senedd Cymru"),
+        ("parl", "UK Parliament"),
+    ]
+
+
+def election_types_choices():
+    qs = (
```

Comment:
Since the election types are hardcoded, there is no need for this query or even the `_get_election_types_choices_for_qs` method, since the `qs` is not used. So you may as well return the list here directly instead